langchain/tools/python/tool.py

```diff
@@ -28,7 +28,7 @@ class PythonREPLTool(BaseTool):
 
     def _run(self, query: str) -> str:
         """Use the tool."""
-        return self.python_repl.run(query)
+        return self.python_repl.run(query.replace('```', ''))
```

Comment:
going to add a flag for this, something like `sanitize_input`

Also probably going to do some `strip()` method in case theres a weird case where this is in a string in the python command